subsys/bluetooth/host/monitor.c

```diff
@@ -325,6 +325,43 @@ static void monitor_log_put(const struct log_backend *const backend,
 	atomic_clear_bit(&flags, BT_LOG_BUSY);
 }
 
+static void monitor_log_process(const struct log_backend *const backend,
+				union log_msg2_generic *msg)
+{
+	struct bt_monitor_user_logging user_log;
+	struct monitor_log_ctx ctx;
+	struct bt_monitor_hdr hdr;
+	static const char id[] = "bt";
+
+	log_output_ctx_set(&monitor_log_output, &ctx);
+
+	ctx.total_len = 0;
+	log_output_msg2_process(&monitor_log_output, &msg->log,
+			       LOG_OUTPUT_FLAG_CRLF_NONE);
+
+	if (atomic_test_and_set_bit(&flags, BT_LOG_BUSY)) {
+		drop_add(BT_MONITOR_USER_LOGGING);
+		return;
+	}
+
+	encode_hdr(&hdr, (uint32_t)log_msg2_get_timestamp(&msg->log),
+		   BT_MONITOR_USER_LOGGING,
+		   sizeof(user_log) + sizeof(id) + ctx.total_len + 1);
```

Comment:
Just making sure this is intentional: `sizeof(id)` would return 3. Did you mean strlen? (same for line 352 and 356).

Reply:
Seems so. it's copied from `monitor_log_put` added by @jhedberg. @jhedberg what do you think?

Reply:
@jhedberg Could you take a look?

Reply:
Yes, it's intentional since we want the null byte at the end included. `strlen(id) + 1` would give you the same, but that has an added runtime penalty that `sizeof(id)` doesn't have.